dashboard/src/components/Tabs/Builds/BuildCards.tsx

```diff
@@ -35,13 +33,14 @@ const ErrorsSummaryBuild = ({
         <DumbSummary summaryHeaders={summaryHeaders}>
           {summaryBody?.map(row => {
             return (
-              <SummaryItem
+              <MemoizedSummaryItem
                 key={row.arch.text}
                 arch={{ text: row.arch.text }}
+                diffFilter={diffFilter}
                 onClickCompiler={value =>
-                  toggleFilterBySection(value, 'compilers')
+                  toggleFilterBySection(value, 'compilers' as T)
```

Comment:
ditto about as

Reply:
same for as, fix it in this commit so it will never exists in the git commit history